src/styles/bloks/card.scss

```diff
@@ -0,0 +1,100 @@
+@use '../utils/variables' as *;
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: $card-width;
+  height: $card-height;
+  background-color: white;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-left: 19px;
+    object-fit: fill;
+  }
+
+  &__description {
+    margin-left: $card-margin;
+    margin-right: $card-margin;
+    font-size: 12px;
+    text-align: left;
+    margin-top: 36px;
+    font-family: $main-font;
```

Comment:
The variable `$card-margin` and `$main-font` are used here. Ensure these variables are defined in the `variables` file. If they are not, it will result in a compilation error.